tests/schemas/api_v3/transactionsDryRun.schema.js

```diff
@@ -77,5 +84,8 @@ module.exports = {
 	dryrunTransactionInvalidResponseSchema: Joi.object(
 		dryrunTransactionInvalidResponseSchema,
 	).required(),
-	metaSchema: Joi.object().required(),
+	metaSchema: Joi.object().optional(),
```

Comment:
Why `optional`?

Reply:
Because meta is an object and currently we are filtering out empty objects from the response.

Reply:
It shouldn't filter it out if we have the envelop set in the method definition 🤔

Reply:
As discussed, let's create a separate issue to address this. Resolving the comment for now.